### ISSUE TITLE
<Select> on:change does not work

### ISSUE THREAD

**Issue by risalfajar** [hperrin/svelte-material-ui#349]
**Describe the bug**
on:change never called when new value is selected

**To Reproduce**
```
<Select variant='outlined' bind:value={editedUser.currentProject} on:change={() => console.log('change called')} label='Proyek'>
    <Option value='name 1'>name 1</Option>
    <Option value='name 2'>name 2</Option>
 </Select>
```

**Expected behavior**
on:change triggered and console.log() should be called

**Desktop (please complete the following information):**
 - OS: Pop OS! 21.04
 - Browser Brave
 - Version 1.30.89 Chromium: 94.0.4606.81 (Official Build) (64-bit)

**Comment by hperrin** [comment]
Yes, the Select component doesn't actually use a native input element, so it doesn't fire a `change` event.

**Comment by hperrin** [comment]
I'll leave this open to track properly documenting this.

**Comment by hperrin** [comment]
Thank you for filing this issue!

I'm going to close this issue for two reasons:

1. v9 has just been released, and I'm going to close all previous issues to be revalidated against v9.
2. SMUI is about to be migrated to the [SciActive Forge](https://forge.sciactive.com/sciactive).

**If you feel this issue should be reopened, please validate that it still applies to the newest version of SMUI (v9), then reopen it.**

As part of the migration to SciActive Forge, this repo **will remain** on GitHub as a mirror, but certain GitHub features will be disabled. This might include GitHub Issues, and I'd like to clean up all of the existing issues to make sure only issues that still apply to v9 get migrated. Thank you so much for your participation!
